### PR TITLE
Add AA jobs as a blocker for GPDB6 release candidate

### PR DESCRIPTION
**Pull request by khannaekta**
As part of commit fa63e7a, the AA jobs were moved out of the release
candidate check. Reverting it back.

Co-authored-by: Bradford Boyle <bboyle@pivotal.io>

### CODE REVIEW

File: concourse/pipelines/templates/gpdb-tpl.yml [hunk 1]
```diff
@@ -1795,9 +1796,9 @@ jobs:
     file: madlib_ci/concourse/tasks/madlib_test_gppkg.yml
     image: gpdb6-centos6-test
     params:
-      TEST_OS: centos
+      TEST_OS: centos6
       ORCA: "off"
-      DBVER: gpdb6-assert
```
Comment (amilkh) [inline concourse/pipelines/templates/gpdb-tpl.yml]: if you need `DBVER` to be different on `6X_STABLE` and `6X_STABLE_without_asserts` you could define a variable in gpdb_6X_STABLE{,_without_asserts}-ci-secrets.prod.yml

File: concourse/pipelines/templates/gpdb-tpl.yml [hunk 2]
```diff
@@ -651,6 +651,7 @@ resources:
   type: git
   source:
     branch: {{madlib-git-branch}}
+    tag_filter: {{madlib-git-tag-filter}}
```
Comment (amilkh) [inline concourse/pipelines/templates/gpdb-tpl.yml]: Did you set `6X_STABLE` and `6X_STABLE_without_asserts` with this change? Maybe it's better to wait until this PR is merged so the two are consistent, see #gpdb-ci.
https://prod.ci.gpdb.pivotal.io/teams/main/pipelines/state-checker/jobs/6X_STABLE-check/builds/123
https://prod.ci.gpdb.pivotal.io/teams/main/pipelines/state-checker/jobs/6X_STABLE_without_asserts-check/builds/124

Comment (khannaekta) [reply concourse/pipelines/templates/gpdb-tpl.yml]: No we haven't set it yet. We will merge in the PR and then update  `6X_STABLE` and `6X_STABLE_without_asserts`

File: concourse/pipelines/templates/gpdb-tpl.yml (hunk 1 above)
Comment (khannaekta) [reply concourse/pipelines/templates/gpdb-tpl.yml]: Thanks for the suggestion, but currently this variable is used to download dependencies for MADlib 1.16, which are required for both with/without assert builds. Therefore, keeping it consistent.

File: concourse/pipelines/templates/gpdb-tpl.yml (hunk 2 above)
Comment (amilkh) [reply concourse/pipelines/templates/gpdb-tpl.yml]: Okay good, looks like the state-checker failures are due to something else.

**Comment by amilkh** [comment]
Great! looks like state-checker is green now also. I've delete the branch.

**Pull request merged**